tokio/src/task/join_set.rs

```diff
@@ -374,6 +374,26 @@ impl<T: 'static> JoinSet<T> {
         while self.join_next().await.is_some() {}
     }
 
+    /// Awaits the completion of all tasks on this `JoinSet`, returning a Vector of their results.
```

Comment:
Either say "vector" or "Vec". There's no such thing as a "Vector".
```suggestion
    /// Awaits the completion of all tasks in this `JoinSet`, returning a vector of their results.
```